libs/components/code-examples/src/lib/modules/data-manager/data-manager/basic/example.component.ts

```diff
@@ -82,6 +92,13 @@ export class DataManagerBasicExampleComponent {
         ],
       }),
     });
+
+    this.#dataManagerSvc
+      .getDataSummaryUpdates(SOURCE_ID)
+      .pipe(takeUntilDestroyed(this.#destroyRef))
+      .subscribe((summary) => {
+        this.recordCount = summary.itemsMatching;
```

Comment:
It'll be easy enough to make these signals, which is what we want our consumers to be using. 👍 
```
protected readonly recordCount = toSignal(
  this.#dataManagerSvc
    .getDataSummaryUpdates(SOURCE_ID)
    .pipe(map((summary) => summary.itemsMatching)),
  { initialValue: 0 },
);
```

Reply:
updated